### PR TITLE
Minor changes from readthrough

### PR DESCRIPTION
**Pull request by tobyhodges**
I will only put small changes into this PR. Anything major that I think needs doing will go in a separate branch to avoid cross-contamination.

Currently WIP. Progress:

- [x] Introduction
- [x] Basic Features
- [x] Callbacks
- [x] Objects and Classes
- [x] HTML and CSS
- [x] Manipulating Pages
- [x] Dynamic Pages
- [x] Visualizing Data
- [x] Promises
- [x] Interactive Sites
- [x] Managing Data
- [x] Creating a Server
- [x] Testing
- [x] Capstone Project
- [x] Conclusion

### CODE REVIEW

File: _en/vis.md [hunk 1]
```diff
@@ -145,17 +170,13 @@ We can fill in the options argument to `vegaEmbed` to turn those off:
...
+{: title="src/vis/vega-disable-controls.html"}
 
 We now have the visualization we wanted:
 
 <figure id="f:vis-disable-controls">
   <figcaption>Without Controls</figcaption>
-<<<<<<< HEAD
-  <img src="../../files/vis-disable-controls.png" />
-=======
   <img id="f:vis-disable-controls" src="../../files/vis-disable-controls.png" alt="Without Controls" />
```
Comment (gvwilson) [inline _en/vis.md]: The `id` and `alt` should be removed here - they go in the `figure` element and in `figcaption` respectively.

File: _en/vis.md [hunk 2]
```diff
@@ -196,15 +217,11 @@ which is set to `"average"`:
...
 ```
-{: title="src/viz/vega-aggregate-points.html"}
+{: title="src/vis/vega-aggregate-points.html"}
 
 <figure id="f:vis-aggregate-points">
   <figcaption>Aggregating and Using Points</figcaption>
-<<<<<<< HEAD
-  <img src="../../files/vis-aggregate-points.png" />
-=======
   <img id="f:vis-aggregate-points" src="../../files/vis-aggregate-points.png" alt="Aggregating and Using Points" />
```
Comment (gvwilson) [inline _en/vis.md]: The `id` and `alt` should be removed here - they go in the `figure` element and in `figcaption` respectively.

**Comment by gvwilson** [comment]
Great work so far - thanks much.

**Comment by tobyhodges** [comment]
@gvwilson please take a look at changes to promises made in https://github.com/software-tools-in-javascript/js-vs-ds/pull/79/commits/8e85809c013b4f2f544cfd581b518d9cb039a4bd - I added a few more sentences and I'd like to make sure that you approve :)

**Comment by tobyhodges** [comment]
Closes #86 

This is ready, but renders one of your PRs from today redundant. Sorry, @gvwilson

**Comment by gvwilson** [comment]
Very happy to have my PRs made redundant :-)

**Pull request merged**